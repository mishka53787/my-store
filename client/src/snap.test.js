import React from 'react';
import { render } from '@testing-library/react';
import Login from './compenents/login';

test('Login component snapshot', () => {
  const { asFragment } = render(<Login />);
  expect(asFragment()).toMatchSnapshot();
});
