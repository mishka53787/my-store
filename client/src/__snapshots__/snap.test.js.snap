// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      User Login
    </h1>
    <form>
      <input
        name="usernameOrEmail"
        placeholder="Username or Email"
        type="text"
        value=""
      />
      <input
        name="password"
        placeholder="Password"
        type="password"
        value=""
      />
      <button
        type="submit"
      >
        Login
      </button>
    </form>
  </div>
</DocumentFragment>
`;
